COM-12827-Automate release process completely with semantic commits of GHA. (#237)
fluendo/fluster: README.md

3. Install the [git hook](https://github.com/fluendo/fluster/blob/master/scripts/install_git_hook.sh) that will run for every
   commit to ensure it works before pushing. [About git
   hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks).
4. Follow the Conventional Commits guidelines for commit messages. This ensures
   your commits follow the correct format for versioning and changelog generation.
   You can learn more about Conventional Commits here: [Conventional Commits Specification](https://www.conventionalcommits.org/en/v1.0.0/).
    - Example commit messages:
        - feat: add new feature
        - fix: fix bug in feature
        - docs: update documentation
    - The pre-commit hook will automatically check that your commit messages
      follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
      Any commit that doesn't adhere to the format will be rejected.
    - Please correct the following errors:
        - Expected value for type from: build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test.

> 5. Modify the code. Make sure the git hook is properly checking that the basic

Reviewer: ```suggestion
5. Modify the code. Make sure git hooks are properly checking that the basic
```

> 5. Modify the code. Make sure git hooks are is properly checking that the basic

Reviewer: ```suggestion
5. Modify the code. Make sure git hooks are properly checking that the basic
```

Author: Done, thx

   vpx-tools aom-tools
   ```

6. Create a new PR with your changes.

> 7. Make sure the GitHub Actions is running and its result is a pass.

Reviewer: ```suggestion
7. Make sure GitHub Actions jobs are launched and all results are a pass.
```
